metpy/calc/indices.py

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2008-2015 MetPy Developers.
+# Distributed under the terms of the BSD 3-Clause License.
+# SPDX-License-Identifier: BSD-3-Clause
+"""Contains calculation of various derived indicies."""
+import numpy as np
+
+from ..calc import mixing_ratio, saturation_vapor_pressure
+from ..calc.tools import get_layer
+from ..constants import g, rho_l
+from ..package_tools import Exporter
+from ..units import units
+
+exporter = Exporter(globals())
+
+
+def precipitable_water(dewpt, p, top=400 * units('hPa')):
+    r"""Calculate precipitable water through the depth of a sounding.
+
+    Default layer depth is sfc-400 hPa. Formula used is from p. 170 in
+    An Introduction to Atmospheric Thermodynamics by Anastasios A. Tsonis
+    (2007):
+
+    \begin{align}
+    \frac{1}{pg} \int\limits_0^d x \,dp
+    \end{align}
+
+    Tsonis, A. A., 2008: An introduction to atmospheric thermodynamics.
```

Comment:
This should go in `docs/references.rst`, with say a key of [Tsonis2008], and then you should include it here as `[Tsonis2008]_, p. 170`.